src/tiled/mainwindow.h

```diff
@@ -211,6 +215,7 @@ private slots:
     QAction *mViewsAndToolbarsAction;
     QAction *mShowObjectTypesEditor;
 
+    QAction *mResetDefaultLayout;
```

Comment:
Should be named `mResetToDefaultLayout` or simply `mResetLayout`, since it's not resetting the default layout.